From 6ad4b17d428d9a13670eb4ea0db52664d7a53aaa Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Thu, 18 Jan 2024 11:26:30 +0100
Subject: [PATCH 0574/1695] dts: stm32: set fast pull down on BUCK6 on
 stm32mp257f-ev1

Enable fast pull down on BUCK6 to improve DDR sub system
power supply shutdown.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 780c6fc54d6642995ea1dcd4f60f3947dc396b17 dts: stm32: set fast pull down on BUCK6 on stm32mp257f-ev1]
Change-Id: I6ff83995ed92d4db619a8ec15a17459bff4e9473
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/352306
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 663c03cb5..e86bf1d78 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -173,6 +173,7 @@
 				regulator-min-microvolt = <1200000>;
 				regulator-max-microvolt = <1200000>;
 				regulator-always-on;
+				regulator-pull-down;
 				st,pwrctrl_sel = <1>;
 				st,pwrctrl_enable;
 
-- 
2.47.2

